test/specs/components/Form/FormField-test.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react'
+import { isConformant } from 'test/specs/commonTests'
+
+import { implementsShorthandProp } from '../../commonTests'
+import { Button, RadioGroup, Input, Text, FormField } from '../../../../src/'
+import { mountWithProvider } from '../../../utils'
+import Slot from '../../../../src/components/Slot/Slot'
+
+const formFieldImplementsShorthandProp = implementsShorthandProp(FormField)
+
+describe('FormField', () => {
+  isConformant(FormField)
+  formFieldImplementsShorthandProp('label', Text)
+  formFieldImplementsShorthandProp('message', Text)
+  formFieldImplementsShorthandProp('control', Slot)
+
+  it('renders the control provided in the controlType prop', () => {
```

Comment:
now this could be renamed :)

Reply:
This will be renamed, no need for the second now that the control is a regular shorthand :)